MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst
+prune workalendar/tests*
```

Comment:
> I can't see any use-case where you would need to import test packages at runtime.

When your python package is packaged for Debian, they want to run the tests to see if the package is correctly packages.

    py.test workalendar